src/CarbonAware.DataSources/CarbonAware.DataSources.Json/src/Configuration/JsonDataSourceConfiguration.cs

```diff
@@ -10,7 +10,7 @@ public class JsonDataSourceConfiguration
 {
     private const string BaseDirectory = "data-sources/json";
     private const string DefaultDataFile = "test-data-azure-emissions.json";
-    private const string DirectoryRegExPattern = @"^[-/a-zA-Z_\d ]*$";
+    private const string DirectoryRegExPattern = @"^[-\\/a-zA-Z_\d ]*$";
```

Comment:
@juzuluag FYI I had to add this to get some of the tests to pass in Windows, since `Path.GetDirectoryName` flips the slash automatically.

Reply:
> @juzuluag FYI I had to add this to get some of the tests to pass in Windows, since `Path.GetDirectoryName` flips the slash automatically.

Who use windows ;-)